tailwind.config.ts

```diff
@@ -23,6 +23,8 @@ module.exports = {
         accent: {
           DEFAULT: 'rgb(var(--color-accent))',
         },
+        black: 'rgb(var(--color-black))',
+        overlay: 'rgb(var(--color-overlay))',
```

Comment:
What was the purpose behind adding the overlay?

Reply:
The background overlay when the link is clicked and opens the Drawer.
![133200](https://github.com/Tascurator/tascurator-frontend/assets/77369174/9636bcb5-f9b5-4d37-9659-8770f8888174)

Reply:
I understand! Thank you for your answer!